crates/symbolicator-service/src/config.rs

```diff
@@ -375,6 +375,25 @@ pub struct Config {
     #[serde(with = "humantime_serde")]
     pub connect_timeout: Duration,
 
+    /// The time window for the host deny list.
+    ///
+    /// Hosts will be put on the deny list if a certain number of downloads
+    /// fail within this time window.
+    #[serde(with = "humantime_serde")]
+    pub deny_list_time_window: Duration,
```

Comment:
Instead of having these in the root, we could maybe group them together in a `downloads` category. Same applies for other things like the `connect_timeout` / `streaming_timeout` above/below, though they need to be backwards compatible if we decide we want to move them.

Reply:
Yeah, I think that's a good idea, but I would like to do it in a separate PR.